templates/includes/sidebar.html

```diff
@@ -1,4 +1,11 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li><a href="{% url 'taxi:driver-detail' pk=user.id %}">User: {{ user.username }}</a></li>
+    <li><a href="{% url 'logout' %}">Logout</a></li>
+  {% else %}
+    <a href="{% url 'login' %}">Login</a>
```

Comment:
The `Login` link is not enclosed within a `<li>` tag, unlike the other links. Consider wrapping it in a `<li>` tag for consistent styling and structure.